### DISCUSSION TITLE
Skip connections for Graph Attention Network

### DISCUSSION THREAD

**Discussion by PietroNardelli** [stellargraph/stellargraph#1983] [Q&A]
Hello,

I am trying to implement a GAT network for inductive learning (something similar to what is done for the PPI dataset in the original paper) and I was wondering if skip connections are already implemented inside the stellargraph GAT network. If not, where should the skip connections go?

Thanks a lot!

**Comment by ecastillot** [comment]
I am wondering the same question. Did you find any answer?

**Reply by diamondspark** [reply]
https://docs.dgl.ai/en/0.6.x/_modules/dgl/nn/pytorch/conv/gatconv.html

You might find this implementation helpful.
